.evergreen/publish-dev-release.sh

```diff
@@ -13,9 +13,22 @@ if [[ "${EVERGREEN_BRANCH_NAME}" != "main" ]]; then
   exit 0;
 fi
 
+JSON_CONTENT=$( jq -n \
+  --arg id "$DEV_VERSION_IDENTIFIER" \
+  --arg key "${EVERGREEN_REVISION}_${EVERGREEN_REVISION_ORDER_ID}" \
+  '{version: $id, bucket_key_prefix: $key}'
+)
+
 URL="https://mciuploads.s3.amazonaws.com/${EVERGREEN_PROJECT}/compass/dev/$1"
```

Comment:
NIT: is it really necessary to parametrize this script, will we ever use it to produce multiple files?

Reply:
no, but i did that so that when we use script in [functions.yml](https://github.com/mongodb-js/compass/blob/main/.evergreen/functions.yml#L455), user would know the expected file name and then use same in the function to upload the file to s3, or else filename is spread.